roles/helm_install/tasks/download.yml

```diff
@@ -0,0 +1,24 @@
+- name: 'Download helm archive'
```

Comment:
Would it make sense to do all of this in some temp directory generated through Ansible, rather than top `/tmp`/?

Reply:
Done